cogs/moveusers.py

```diff
@@ -4,10 +4,11 @@
 # Command that moves one user in a voice channel into another voice channel specified by the user
 class MoveUsers(commands.Cog):
 
+# Function dedicated to moving a specified user to a specified channel
```

Comment:
Nit: Some comments have periods and some don't. Could we keep it consistent and do either one? Thanks!